src/ResourceManager/Compute/ChangeLog.md

```diff
@@ -23,6 +23,8 @@
     - Added `Export-AzureRmLogAnalyticThrottledRequests` cmdlet
 * `New-AzureRmVMSS` prints connection strings in verbose mode.
 * `New-AzureRmVmss` supports public IP address, load balancing rules, inbound NAT rules.
+* `Get-AzureRmVmssDiskEncryptionStatus` reports encryption state of scale set using disk metadata
```

Comment:
@ejarvi would you mind pulling in the latest changes from the `preview` branch? The contents you added should be under the `## Current Release` header and everything else should be under the `## Version 4.5.0` header

Reply:
@cormacpayne updated